antsibull/build_changelog.py

```diff
@@ -527,6 +527,9 @@ def _get_porting_guide_bytes(changelog: Changelog) -> bytes:
                 " fully-qualified collection name (FQCN) in your playbooks as the explicit and"
                 " authoritative indicator of which collection to use as some collections may"
                 " contain duplicate module names."
+                " You can search the"
+                " `index <https://docs.ansible.com/ansible/latest/collections/index_module.html>`_"
```

Comment:
The link title should contain more mords to be more useful:
```suggestion
                " `index of all modules <https://docs.ansible.com/ansible/latest/collections/index_module.html>`_"
```

Reply:
@felixfontein the "of all modules" is on the next line actually…

s/mords/words/
And please,  _@felixfontein, don't bite your friends!_ :laughing:

Reply:
Link titles should be complete and useful, [not just for accessability reasons](https://github.com/ansible/ansible/issues/73510).

And yes, I saw that the text is in the next line, I assumed that whoever accepts this suggestion will manage to remove it there.

Reply:
@felixfontein that's actually how I originally wrote it until linter complained about it :(
See https://github.com/ansible-community/antsibull/pull/252/commits/1f3cb60148ff7d600203a89425c77db12179e8ce

Reply:
You can split it like this:
```
" `index of all modules"
" <https://docs.ansible.com/ansible/latest/collections/index_module.html>`_"
```